ext/console/02_console.js

```diff
@@ -658,7 +658,7 @@ let circular;
 function handleCircular(value, cyan) {
   let index = 1;
   if (circular === undefined) {
-    circular = new Map();
+    circular = new SafeMap();
     MapPrototypeSet(circular, value, index);
```

Comment:
FWIW this could be simplified to
```suggestion
    circular.set(value, index);
```